ci/tests/test_01_functional/test_04_api/__init__.py

```diff
@@ -0,0 +1,98 @@
+import os
+from ci.tests.test_01_functional.test_04_api.apitest\
+ import APITestCase
+from container_pipeline.lib import dj
+from container_pipeline.models.pipeline import Project,\
+    Build, BuildPhase
+import json
+from django.utils import timezone
+
+class ApiV1TestCase(APITestCase):
+    """
+    Module to test api v1.
+    """
```

Comment:
define node here @mohammedzee1000 

otherwise the tests will be executed on the controller node and thus you are getting import error for django